### ISSUE TITLE
New rule to forbid to use `expect(find*(...)).toBeTruthy();`

### ISSUE THREAD

**Issue by retyui** [testing-library/eslint-plugin-testing-library#805]
### Name for new rule

your-name

### Description of the new rule

False Positives check, that need to be forbidden:

```tsx
find*(...) // Promise<T>

expect(promise).toBeDefined(); // true
expect(promise).toBeTruthy(); // true
```

### Testing Library feature

`find*`

### Testing Library framework(s)

dom & react-native

### What category of rule is this?

Suggests an alternate way of doing something

### Code examples

```tsx
expect(find*(...)).toBeDefined(); // bad
expect(find*(...)).toBeTruthy(); // bad
```

### Do you want to submit a pull request to make the new rule?

No

**Comment by Belco90** [comment]
Hi @retyui. You can achieve this using [the `eslint-plugin-jest/no-restricted-matchers` rule](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-restricted-matchers.md).
